src/main.js

```diff
@@ -1,3 +1,4 @@
+import 'regenerator-runtime/runtime.js'
```

Comment:
I think this is no longer required. But we can take this up in the next release. Not a blocker/issue